home-page.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+<link rel="stylesheet" href="home-page-style.css">
+</head>
+<body>
+  <header class="header">
+<div id="recipe-page">Recipe</div>
```

Comment:
You make this behave as an anchor tag through JavaScript. Why not just make it in HTML?
```suggestion
<a href="#">Recipe</a>
```
then you can add `class` and do the styling on it, instead on the `id`